src/libs/workspace-utils.ts

```diff
@@ -29,6 +29,11 @@ interface BaseWorkspaceInfo {
   createdDate: string;
   createdBy: string;
   lastModified: string;
+  bucketName?: string;
```

Comment:
`bucketName` is only for Google workspaces and should not be part of BaseWorkspaceInfo.

Reply:
I'm guessing that the root cause for adding this here may be some places that incorrectly use `BaseWorkspace` as a type for some input. `BaseWorkspace` really shouldn't be exported from this module, it's only for defining fields common to `Azure/GoogleWorkspace`. All consumers should use either `WorkspaceWrapper` or one of the `GoogleWorkspace`/`AzureWorkspace` types.

Reply:
:lightbulb: